marmoset/dhcp/dhcp_config.py

```diff
@@ -0,0 +1,68 @@
+from .isc_dhcp_ldap_config import ISCDhcpLdapConfig
+from marmoset import validation
+import socket
+
+
+class DhcpConfig:
+    def __init__(self, mac, ip_address, gateway=None, networkmask=None):
+        self.additional_statements = {}
```

Comment:
this initialize only a dictionary for additonal_statements... here is nothing added